src/firefly/js/tables/TableUtil.js

```diff
@@ -1569,11 +1569,21 @@ export function splitVals(values='') {
 
 export function parseError(error) {
     const message = error?.message ?? error;
+    const colonRegex = /(.+?):(.+)/;
+
     if (error?.cause) {
-        const [_, type, cause] = error?.cause.match(/(.+?):(.+)/) || [];
+        const [_, type, cause] = error.cause.match(colonRegex) || [];
         return {message, type, cause};
     } else {
-        const [_, error, cause] = message?.match(/(.+?):(.+)/) || [];     // formatted error messages; 'error:cause'
+        // Check if message contains URLs or JSON - if so, don't split
+        const hasUrl = /https?:\/\//.test(message);
+        const hasJson = /[{[]/.test(message) && /[}\]]/.test(message);
+        if (hasUrl || hasJson) {
+            return {message: 'Table Error', cause: message};
+        }
```

Comment:
This is fine for now.  But, we may need to formalize how table errors are returned from the server. Relying on checks for 'https' or JSON-style braces may be unreliable, as those values could appear in the actual error message content.

Reply:
Agreed. I could conjure up an ugly regex to escape the `:` in urls. But for JSON it was getting way too complicated. In an ideal scenario, server should return the message, cause as 2 entities or use something other than `:`, if need to return a single string containing both.